packages/expo-background-task/ios/BackgroundTaskConstants.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+import Foundation
+
+public class BackgroundTaskConstants: NSObject {
```

Comment:
```suggestion
public struct BackgroundTaskConstants {
```

can just use struct and not necessarily derive NSObject